### ISSUE TITLE
Initial Registration is failing after triggering deregister normal

### ISSUE THREAD

**Issue by BG00818733** [aligungr/UERANSIM#608]
Hi @aligungr,

**Setup Details**
Open5GS v2.5.8 running on Kubernetes deployment(EKS Cluster) with UERANSIM(EC2 Instance).
**Use Case:**
We are able to do the UE Registration and PDU session successfully. After that we are triggering Normal deregistration using below command.
 ./build/nr-cli imsi-208930000000001 -e "deregister normal"

Expected behavior from nr-ue process is that UE should register again to the core network, but in our case it is failing with cause "![image](https://user-images.githubusercontent.com/122294196/220956251-d9f78c3b-0261-4ed4-a27b-502c7058c9ba.png)".
Below is the Wireshark PCAP with detailed packet capture.

PCAP: [test_pr2.zip](https://github.com/aligungr/UERANSIM/files/10815487/test_pr2.zip)

Please help us with the fix. Additionally, we want to know the procedure for triggering the Periodic Registration.
Your response will be appreciated.

Thanks & Regards,
Bharat Gupta

**Comment by BG00818733** [comment]
Hi @aligungr 

We triggered the Periodic registration, but still we are getting the same issues as mentioned above. We debugged further and found that nr-ue utility is unbale to encode the SD value in Requested NSSAI which is getting encoded properly while bringing up binary. Below is the pacp capture for your reference.

[test_periodic_reg_2_2.zip](https://github.com/aligungr/UERANSIM/files/10839265/test_periodic_reg_2_2.zip)

Your inputs will be highly appreciated.

Thanks & Regards,
Bharat Gupta
